### PR TITLE
New package PKGBUILD

### PR DESCRIPTION
**Pull request by liborw**

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,75 @@
+pkgname="ros-melodic-franka-control"
+pkgver="0.6.0"
+pkgrel=2
```
Comment (acxz) [inline PKGBUILD]: why is pkgrel 2, it should be 1 since this is the first one

File: fix_pthread.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+diff --unified --recursive --text franka_ros-release-release-melodic-franka_control-0.6.0-1/CMakeLists.txt franka_ros-release-release-melodic-franka_control-0.6.0-1.new/CMakeLists.txt
+--- franka_ros-release-release-melodic-franka_control-0.6.0-1/CMakeLists.txt	2019-07-11 09:54:40.000000000 +0200
```
Comment (acxz) [inline fix_pthread.patch]: I dont think this patch works. The file name to be patched has to be the same.

**Review by acxz — CHANGES_REQUESTED** [review]

File: fix_pthread.patch (hunk 2 above)
Comment (liborw) [reply fix_pthread.patch]: Yeah forgot to remove the patch it was not used, fixed.

Comment (acxz) [reply fix_pthread.patch]: @liborw sorry for not personally telling you this but we have shifted our operations over to the [ros-noetic-arch](https://github.com/ros-noetic-arch) github org. See: https://discourse.ros.org/t/who-is-using-ros-on-archlinux/388/66?u=acxz
Feel free to open up the package requests there

**Pull request closed without merging**